<ui version="4.0" >
 <class>dEditLang</class>
 <widget class="Q3MainWindow" name="dEditLang" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>299</width>
    <height>185</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Language</string>
  </property>
  <widget class="QWidget" name="widget" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>299</width>
     <height>185</height>
    </rect>
   </property>
   <layout class="QGridLayout" >
    <item row="0" column="0" >
     <widget class="QTabWidget" name="tabWidget11" >
      <widget class="QWidget" name="tab" >
       <attribute name="title" >
        <string>Common</string>
       </attribute>
       <layout class="QGridLayout" >
        <item row="0" column="0" >
         <layout class="QHBoxLayout" >
          <item>
           <layout class="QVBoxLayout" >
            <item>
             <widget class="QLabel" name="textLabel1" >
              <property name="text" >
               <string>Tag:</string>
              </property>
              <property name="wordWrap" >
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="textLabel2" >
              <property name="text" >
               <string>Name:</string>
              </property>
              <property name="wordWrap" >
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="textLabel3" >
              <property name="text" >
               <string>Translation file:</string>
              </property>
              <property name="wordWrap" >
               <bool>false</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" >
            <item>
             <widget class="QLineEdit" name="eTag" />
            </item>
            <item>
             <widget class="QLineEdit" name="eName" />
            </item>
            <item>
             <widget class="QLineEdit" name="eTrFile" />
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item row="1" column="0" >
         <spacer>
          <property name="orientation" >
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType" >
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>21</width>
            <height>141</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="5" margin="5" />
 <pixmapfunction>rcIcon</pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>Q3MainWindow</class>
   <extends>QWidget</extends>
   <header>q3mainwindow.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <includes>
  <include location="local" >mdtree.h</include>
  <include location="local" >cfgform.h</include>
  <include location="local" >mainform.h</include>
 </includes>
 <resources/>
 <connections/>
</ui>
